### ISSUE TITLE
Video not accessible when using VoiceOver on iPhone and iPhoneUseNativeControls: true

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-media#307]
### Subject of the issue
The video area of the Media component cannot be navigated to with VoiceOver on iPhone when using `"iPhoneUseNativeControls": true`. This setting forces the use of iPhone's native controls. VoiceOver will skip over the video and go directly to the Transcript button (assuming it's enabled).

I am not sure if this is a MediaElement.js bug, but the same issue happens with v7.0.5 in #299 

### Your environment
* FW 5.41.5
* Media 6.7.3

### Steps to reproduce
1. Configure a Media component to use `"iPhoneUseNativeControls": true`
2. On an iPhone, enable VoiceOver at Settings > Accessibility > VoiceOver.
3. Load the page in Safari
4. Swipe forward until you come to the Media component.

### Expected behaviour
You should be able to focus on the video and play it.

### Actual behaviour
VoiceOver navigation skips over the video entirely.
